main.go

```diff
@@ -48,10 +42,10 @@ func main() {
 	//Generate a test suite based on configuration settings
 	testSuite := new(testStrategies.TestSuite)
 	testSuite.BuildTestSuite(configurationSettings)
-	numTestCases := len(testSuite.TestCases) //convenience variable
+	numTestCases := len(testSuite.TestDefinitions) //convenience variable
 
```

Comment:
See struct consolidations in commonToAllTestStrategies.go. TestCases is used to unmarshal the testSuite definition XML, and TestDefinitions contains the test definitions.